03-task-manager/starter/quizAnswers2.txt

```diff
@@ -0,0 +1,13 @@
+1. In this lesson, you created a middleware function called asyncWrapper. Why?
+    asyncWrapper wrapper is used to handle async errors thrown by the controllers.
```

Comment:
That's right! It helps us reduce repetition, since this way we don't have to write a try/catch block in every route handler and we can just have a centralized way to handle all errors.